R/aglm.R

```diff
@@ -11,6 +11,7 @@
 #' @param qualitative_vars_OD_only A list of indices or names for specifying which columns are qualitative and need only O-dummy representations.
 #' @param quantitative_vars A list of indices or names for specyfying which columns are quantitative.
 #' @param use_LVar A boolean value which indicates whether this function uses L-variable representations or not.
+#' @param extrapolation A type how extrapolate for left boundary. ('flat' or 'default')
```

Comment:
- would like to add values are character
- maybe the right side is also extrapolate